snuba/migrations/migrations.py

```diff
@@ -0,0 +1,91 @@
+import logging
+
+from datetime import datetime
+from enum import Enum
+from importlib import import_module
+from typing import Optional, Sequence, Tuple
+
+from snuba.clusters.cluster import ClickhouseClientSettings, get_cluster, CLUSTERS
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations.operations import Operation
+
+
+logger = logging.getLogger("snuba.migrations")
+
+LOCAL_TABLE_NAME = "migrations_local"
+DIST_TABLE_NAME = "migrations_dist"
+
+# Currently only local migrations are supported
+TABLE_NAME = LOCAL_TABLE_NAME
+
+
+class App(Enum):
+    SYSTEM = "system"
+    SNUBA = "snuba"
+
+
+class Status(Enum):
+    NOT_STARTED = "not_started"
```

Comment:
We are never using this state. What will it mean ?

Reply:
It's not one of the states that will be saved to the database, however when we build the tree of migrations to be run, migrations that are not present in the database (and therefore not started yet) will get this status.

Reply:
ok